docs/source/ipython.rst

```diff
@@ -0,0 +1,84 @@
+IPython Integration
+===================
+
+Dask.distributed integrates with IPython in three ways:
+
+1.  You can launch a Dask.distributed cluster from an IPyParallel_ cluster
+2.  You can launch IPython kernels from Dask Workers and Schedulers to assist
+    with debugging
+3.  They both support the common concurrent.futures_ interface
+
+.. _IPyParallel: https://ipyparallel.readthedocs.io/en/latest/
+.. _concurrent.futures: https://docs.python.org/3/library/concurrent.futures.html
+
+
+Launch Dask from IPyParallel
+----------------------------
+
+IPyParallel is IPython's distributed computing framework that allows you to
+easily manage many IPython engines on different computers.
+
+An IPyParallel ``Client`` can launch a ``dask.distributed`` Scheduler and
+Workers on those IPython engines, effectively launching a full dask.distributed
+system.
+
+This is possible with the Client.become_distributed_ method::
+
+   $ ipcluster start
+
+.. code-block:: python
+
+   >>> from ipyparallel import Client
+   >>> c = Client()  # connect to IPyParallel cluster
+
+   >>> e = c.become_distributed()  # start dask on top of IPyParallel
```

Comment:
While we are here, do you have a preference for the name of this method? Should I have called it `.become_dask`?